.github/workflows/testing.yml

```
@@ -8,34 +8,36 @@ on:
   workflow_dispatch:
 
 jobs:
-  run-tests-ce:
-    # We want to run on external PRs, but not on our own internal
-    # PRs as they'll be run by the push to the branch.
-    #
-    # The main trick is described here:
-    # https://github.com/Dart-Code/Dart-Code/pull/2375
-    #
-    # Also we want to run it always for manually triggered workflows.
+  run-tests-on-ubuntu-22-04:
     if: (github.event_name == 'push') ||
       (github.event_name == 'pull_request' &&
         github.event.pull_request.head.repo.full_name != github.repository) ||
       (github.event_name == 'workflow_dispatch')
-
-    # We could replace it with ubuntu-latest after fixing the bug:
-    # https://github.com/tarantool/setup-tarantool/issues/37
-    runs-on: ubuntu-22.04
-
     strategy:
       fail-fast: false
       matrix:
-        golang:
-          - '1.20'
-          - 'stable'
-        tarantool:
-          - '1.10'
-          - '2.8'
-          - '2.10'
-          - 'master'
+        golang: ['1.20', 'stable']
+        tarantool: ['1.10']
+        coveralls: [false]
+        fuzzing: [false]
+    uses: ./.github/workflows/reusable-run.yml
+    with:
+      os: ubuntu-22.04
```

```suggestion
      os: ubuntu-22.04
```
Why the 22.04 is here, but ubuntu-latest in othe jobs? It is better to unify here.

---

Because 1.10 is presented in 22.04, not in ubuntu-latest.

---

https://github.com/tarantool/go-tarantool/actions/runs/17546313637/job/49828488207?pr=458